[IMP] inventory: update 3 step delivery doc (#8837)
odoo/documentation: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/delivery_three_steps.rst

:menuselection:`Inventory --> Configuration --> Settings --> Warehouse`. Note that activating
:guilabel:`Multi-Step Routes` will also activate *Storage Locations*.

> .. image:: delivery_three_steps/delivery-three-steps-multi-step-routes.png

Reviewer: I don't see an improvement here with the new image. Can you help me understand why there was a change?

Process a picking
-----------------

The picking, packing, and delivery orders will be created once the sales order is confirmed.  To
view these transfers, go to :menuselection:`Inventory --> Operations --> Transfers`.
The picking order will be created once the sales order is confirmed. To view the picking, navigate
to the :menuselection:`Inventory app`, and locate the :guilabel:`Pick` task card on the
:guilabel:`Inventory Overview` dashboard.

.. image:: delivery_three_steps/three-step-delivery-transfers.png
   :align: center
   :alt: Ready status for the pick operation while the packing and delivery operations are waiting
         another operation.

The status of the picking will be :guilabel:`Ready`, since the product must be picked from stock
before it can be packed. The status of the packing order and delivery order will both be
:guilabel:`Waiting Another Operation`, since the packing and delivery cannot happen until the
picking is completed. The status of the delivery order will only change to :guilabel:`Ready` when
the packing has been marked :guilabel:`Done`.

The receipt can also be found in the *Inventory* application. In the :guilabel:`Overview` dashboard,

> click the :guilabel:`1 To Process` smart button in the :guilabel:`Pick` kanban card.

Reviewer: I don't know if I agree with removing this additional context around a 3-step operation. **Waiting Another Operation** and **Ready** states are relevant here, why remove them?

Author: Per direct messages - this section was removed due to flow changes from the last update - resolving comment ✅

> Click the :guilabel:`# To Process` button (in this case, :guilabel:`1 To Process`). This reveals the

Reviewer 2: ```suggestion
Click the :guilabel:`# To Process` button, which reveals the
```
Totally up to you, but I don't think mentioning what number is *as* useful in documentation as it is in a eLearning tutorial. Up to you, though!

> .. image:: delivery_three_steps/delivery-three-steps-picking-order.png

Reviewer: I'd revert back to the other screenshot.
Since the old screenshot covers more of the UI, it provides users navigational context as to where we are when we're looking at the **Delivery** smartbutton.

Author: Screenshot has been replaced with a new zoomed-out one, showcasing more of the navigational UI and highlighting key areas. Suggestion implemented ✅

> Once the picking order has been validated, the packing order is ready to process. Click back to the
> :guilabel:`Inventory Overview`, and locate the :guilabel:`Pack` task card on the dashboard.

Reviewer 2: ```suggestion
After validating the picking, the packing order is ready to process. Click back to the
:guilabel:`Inventory Overview`, and locate the :guilabel:`Pack` task card on the dashboard.
```

Author: Suggestion implemented ✅

Click the :guilabel:`# To Process` button (in this case, :guilabel:`1 To Process`). This reveals the
packing order generated from the previously confirmed sales order.

Click on the packing order associated with the sales order, then click on :guilabel:`Validate` to
complete the packing.

> .. image:: delivery_three_steps/delivery-three-steps-packing-order.png

Reviewer: Same here...zooming in like this isn't very helpful I don't think since we're removing surrounding UI elements that give the reader navigational context.

As well, if we're going to take the server hit for storing the screenshot, I'd argue that ideally we super fill these out like a living success model, so we can get the most use out of it (and so would our readers).

Author: Screenshot has been replaced with a new zoomed-out one, showcasing more of the navigational UI and highlighting key areas. Suggestion implemented ✅

Process a delivery
------------------

Once the packing order has been validated, the delivery order is ready to process. Navigate back to
the original sales order to process the delivery by going to :menuselection:`Sales app`, and

> selecting the sales order created previously.

Reviewer 2: In a note admonition block, we can mention that delivery orders can also be accessed by going to Inventory > Operations > Deliveries.

Reviewer 2: Also, I'm not sure if navigating back to the sales order is the go-to instruction here. To understand your perspective here, what was your reasoning for going back to the SO? I understand that doing this completes the sales flow, so the customer can invoice the client next. However, since this doc is in inventory, shouldn't we just go to Inventory > Operations > Delivery Orders, and process the delivery order? 

If you agree with this logic, I'd put the part about going back to the sales app in a note block, instead.

Author: I added a tip admonition to inform the reader that they can also access it that way. But, I'm also keeping the original instruction as I had it, too. The logic of going to the original SO was to show how the Delivery smart button updates with all the transfers as the process is completed, so that all transfers are traceable from one place. Plus, the SO needs to be closed out and invoiced once the delivery is completed.

If I get additional feedback concerning this, however, I will consider changing it.
